### ISSUE TITLE
[object HTMLSpanElement] displayed in multiple locations in the text

### ISSUE THREAD

**Issue by jstnjstn** [hax/atom-elastic-tabstops#30]
[object HTMLSpanElement] displayed in multiple locations in the text.
My package installation is quite minimal so I suppose it is a bug with this package.

**Comment by lillem4n** [comment]
It is. It appears when you have tab and then a space directly after or before it.

**Comment by hax** [comment]
Could you explain this issue in details?

**Comment by lillem4n** [comment]
On a single line, combine tabs and spaces between two other characters.

**Comment by caprilo** [comment]
Yes, I can confirm the same bug in Atom 1.39.1. Just write `a` `[space]` `[tab]` `a` and it will happen.

**Comment by Sod-Almighty** [comment]
Given that this bug has existed unfixed for over a year, I assume this project - interesting and potentially useful as it may be - is dead?

**Comment by hax** [comment]
@Sod-Almighty Sorry, I don't have time to investigate this bug, but PR is always welcome.

**Comment by Sod-Almighty** [comment]
See, this is what is wrong with open source.

"Found a bug? Great! Let me know when you've fixed it!"

If you're not prepared to maintain YOUR EXTENSION, then remove it from the Atom repo and save us all the wasted time.

**Comment by lillem4n** [comment]
@hax thank you for responding and for giving us this extension to begin with.

**Comment by hax** [comment]
@Sod-Almighty 

A close source product which lose the maintaining would make u in worse condition because no one can fix the bugs legally. Open source at least allow others to take it and fix it if it's really important to them.

**Comment by Sod-Almighty** [comment]
True. But if you're not prepared to maintain the project, you should take it down from the Atom repo.

**Comment by iTob191** [comment]
Instead of removing the package from the Atom repo, I suggest removing current (incompatible) versions of Atom from the supported versions range:
https://github.com/hax/atom-elastic-tabstops/blob/956a0a3fd92e3d27364579c70db70c0aea5f119a/package.json#L13
